activesupport/lib/active_support/ordered_options.rb

```diff
@@ -116,6 +116,14 @@ def inspect
       "#<#{self.class.name} #{to_h.inspect}>"
     end
 
+    def to_s
+      to_h.to_s
+    end
+
+    def pretty_print(pp)
+      pp.pp_hash(to_h)
```

Comment:
This is taken right from Ruby:
https://github.com/ruby/ruby/blob/982641939cf709d22dbc060df57cbed31acd3b97/lib/pp.rb#L383-L386